openslides_backend/action/relations.py

```diff
@@ -96,7 +96,7 @@ def perform(self) -> Relations:
                 ],
                 lock_result=True,
             )
-            rels = response[target]
+            rels = response[target] if target in response else {}
```

Comment:
If I call a `topic.create` from the client, the backend tries to fetch mediafiles from the datastore but provides an empty array of ids, which leads to a `KeyError` here. This was the easiest fix for me, feel free to improve it.